src/redux/books/books.js

```diff
@@ -1,65 +1,78 @@
-// Actions types
-import BookstoreAPI from '../../Components/apiConnect';
+import axios from 'axios';
 
+const baseURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/JQgP60QXiQUZk48I3GP2/books/';
 const ADD_BOOK = 'ADD_BOOK';
 const REMOVE_BOOK = 'REMOVE_BOOK';
 const FETCHED_BOOK = 'FETCHED_BOOK';
 
-const fetchBooks = () => async (dispatch) => {
-  const books = await BookstoreAPI.getBooks();
-  if (books) {
-    dispatch({ type: FETCHED_BOOK, books });
-  }
-};
-
-const addBook = ({
-  id, category, bookName, authorName,
-}) => async (dispatch) => {
-  const result = await BookstoreAPI.addBook({
-    id, bookName, authorName, category,
-  });
-  if (result) {
-    dispatch({
-      type: ADD_BOOK,
+// Actions types
+export const fetchBooks = () => async (dispatch) => {
+  const response = await axios.get(baseURL);
+  const books = response.data;
+  const booksData = Object.entries(books);
+  const bookList = [];
+  booksData.forEach(([key, book]) => {
+    const id = key;
+    const {
+      title, author, category,
+    } = book[0];
+    bookList.push({
       id,
+      title,
+      author,
       category,
-      bookName,
-      authorName,
     });
-  }
+  });
+
+  dispatch({
+    type: FETCHED_BOOK,
+    payload: bookList,
+  });
 };
 
-const removeBook = ({ id }) => async (dispatch) => {
-  const result = await BookstoreAPI.removeBook({ id });
-  if (result) {
-    dispatch({
-      type: REMOVE_BOOK,
-      id,
-    });
-  }
+export const addBook = (book) => async (dispatch) => {
+  await fetch(baseURL, {
+    method: 'POST',
+    headers: { 'content-type': 'application/json' },
+    body: JSON.stringify(book),
+  })
+    .then((response) => response.text())
+    .then(
+      () => dispatch({ type: ADD_BOOK, payload: book }),
+    );
+};
```

Comment:
- [ ] Right now, none of the core functionalities of your app work. I cannot remove any of the books that have already been added and I also cannot add a new book. Kindly rectify this issue and make sure your app works as expected before submitting for review 💯 
- Your app:
![Animation](https://user-images.githubusercontent.com/85113567/174224810-d2df591a-7db1-47f0-b8fa-bc83d50c3e0a.gif)